source/reference/oplog-sizing.txt

```diff
@@ -40,55 +41,30 @@ Estimate ``oplog`` Size Needed for Initial Sync
       cluster. If there are multiple shards, the smallest number is the
       minimum ``oplog`` window.
 
-   .. step:: Estimate Copy Rate During Synching
-
-      .. _c2c-step-est-size:
-
-      To gather performance data while synching, start the sync process
-      and monitor how fast data is transferred between clusters.
-      
-      To start syncing, run the :ref:`/start <c2c-api-start>`
-      command.
```

Comment:
(step titles)
For consistency, decide whether you'll want to use sentence case and font treatment throughout. 
I'm not sure we've got good guidelines on procedure step titles. I'm going to try to write some up soon. 

My vote is that you use sentence casing and include the font treatment.

Reply:
Fixed